client/src/components/Creator.js

```diff
@@ -1,8 +1,195 @@
 import React, { Component } from "react";
+import { Container, Row, Col, Button, Card, Table } from "react-bootstrap";
+import { ImCopy, ImTwitter, ImTelegram } from "react-icons/im";
+import { TwitterShareButton, TelegramShareButton } from "react-share";
+import Web3 from "web3";
+import Copy from "clipboard-react";
 
 class Creator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ETHinUSD: "0",
+      aUSDCBalance: "0",
+    };
+  }
+
+  async getBalance() {
+    let tokenAddress = "0xbcca60bb61934080951369a648fb03df4f96263c";
+    let walletAddress = this.props.connectedWallet;
+    if (this.props.provider) {
+      let web3 = new Web3(this.props.provider);
+
+      // The minimum ABI to get ERC20 Token balance
+      let minABI = [
+        // balanceOf
+        {
+          constant: true,
+          inputs: [{ name: "_owner", type: "address" }],
+          name: "balanceOf",
+          outputs: [{ name: "balance", type: "uint256" }],
+          type: "function",
+        },
+        // decimals
+        {
+          constant: true,
+          inputs: [],
+          name: "decimals",
+          outputs: [{ name: "", type: "uint8" }],
+          type: "function",
+        },
+      ];
+
+      // Get ERC20 Token contract instance
+      var contract = new web3.eth.Contract(minABI, tokenAddress);
+      // Call balanceOf function
+      var balance = await contract.methods.balanceOf(walletAddress).call();
+      this.setState({ aUSDCBalance: balance });
+    }
+  }
+
+  componentDidMount() {
+    this.getBalance();
+    fetch(
+      "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=ethereum"
+    )
+      .then((response) => response.json())
+      .then((priceUSD) => {
+        this.setState({ ETHinUSD: priceUSD[0].current_price });
+      });
+  }
+
+  getValue(weiVal) {
+    const val = Web3.utils.fromWei(weiVal, "ether");
+
+    var USD = parseFloat(val) * parseFloat(this.state.ETHinUSD);
+    USD = USD.toFixed(2);
+    return val + " ETH ($" + USD + " USD)";
+  }
+
+  getRealTime(timestamp) {
+    var utcSeconds = parseInt(timestamp);
+    var d = new Date(0); // The 0 there is the key, which sets the date to the epoch
+    d.setUTCSeconds(utcSeconds);
+    return d.toLocaleString();
+  }
```

Comment:
nit: Prefer `const` when the variable is not reassigned after var initalization, otherwise `let`. `var` is not commonly used anymore since it's not block-scoped. More details: https://medium.com/swlh/the-difference-of-var-vs-let-vs-const-in-javascript-abe37e214d66